api/src/data_inclusion/api/inclusion_data/schemas.py

```diff
@@ -51,4 +56,28 @@ class ServiceSearchResult(BaseModel):
 
 
 class DetailedStructure(Structure):
+    doublons: Annotated[
+        list[schema.Structure],
+        Field(description="Doublons connus de la structure"),
+    ]
     services: list[Service]
+
+
+class SourceIdDict(TypedDict):
+    source: str
+    id: str
+
+
+class ListedStructure(Structure):
+    doublons: Annotated[
+        list[schema.Structure],
+        Field(
+            description="Identifiants (source, id) des doublons connus de la structure",
+        ),
+    ]
+
+    @field_serializer("doublons")
+    def serialize_doublons(
+        self, doublons: list[schema.Structure]
+    ) -> list[SourceIdDict]:
+        return [{"source": d.source, "id": d.id} for d in doublons]
```

Comment:
```suggestion
    doublons: Annotated[
        list[SourceIdDict],
        Field(
            description="Identifiants (source, id) des doublons connus de la structure",
        ),
    ]
```

(peut-être quelques ajustements côté orm, mais je ne crois pas)

Reply:
done !